lib/driver.js

```diff
@@ -251,6 +255,31 @@ class AndroidDriver extends BaseDriver {
     await this.initAutoWebview();
   }
 
+  shouldDismissChromeWelcome (opts) {
```

Comment:
what's the point to pass opts here if the method already has access to _this.opts_ ?